README.md

```diff
@@ -11,13 +11,32 @@ It can currently extract:
 - Activated Google services (YouTube, Photos, Maps, News360, Hangouts, etc.)
 - Possible YouTube channel
 - Possible other usernames
-- Public photos
-- Phone models
-- Phone firmwares
-- Installed software
+- Public photos (T)
+- Phones models (T)
+- Phones firmwares (T)
+- Installed softwares (T)
 - Google Maps reviews
 - Possible physical location
 
+The features marked with a **(T)** requiere the target account to have the default setting of `"allow the people you share content with to download your photos and videos. People may still be able to download photos and videos that you've shared with google photos"`. 
+
+Even for a tech-savy person this setting is hard to find, bc of how buried it is, and how much google tried to hide the fact that some albums can be collected regardless of what the account dashboard says:
```

Comment:
I'd personally avoid writing "bc" and other short words. Also I suggest to use capital G for Google on this file.

Reply:
I agree with @outime, and avoiding also terms like "big-ass" :smile:.
Can we also move that Fixing section just before the "Thanks" section ? I think the priority of this project is the offensive part and the fixing part is secondary.

Reply:
Fair enough :p
I'll edit it as soon as possible